samples/python/aruco_detect_board_charuco.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+
+"""aruco_detect_board_charuco.py
+Usage example:
+python aruco_detect_board_charuco.py -w=5 -h=7 -sl=0.04 -ml=0.02 -d=10 -c=tutorial_camera_charuco.yml -v=choriginal.jpg
+-v, --video - Input from video or image file, if omitted, input comes from camera, default=""
+-w - Number of squares in X direction, default="3"
+-h - Number of squares in Y direction", default="3", action="store", dest="h", type=int)
+-sl - Square side length, default="1."
+-ml - Marker side length, default="0.5"
+-d - dictionary: DICT_4X4_50=0, DICT_4X4_100=1, DICT_4X4_250=2,  DICT_4X4_1000=3, ..., default="DICT_4X4_50"
+-ci - Camera id if input doesn't come from video (-v), default="0"
+-c - Input file with calibrated camera parameters, default=""
+-H, --help - show help
```

Comment:
Duplicate of `--help` output.
Duplication must be avoided as it could not be maintained in reliable way.

Reply:
removed duplication